modules/swagger-codegen/src/main/resources/swift3/api.mustache

```diff
@@ -130,7 +130,7 @@ open class {{classname}}: APIBase {
         {{#hasQueryParams}}
         url?.queryItems = APIHelper.mapValuesToQueryItems(values:[
         {{#queryParams}}
-                "{{paramName}}": {{paramName}}{{#hasMore}}, {{/hasMore}}
+                "{{paramName}}": {{paramName}}{{#isEnum}}{{#isContainer}}{{{dataType}}}{{/isContainer}}{{^isContainer}}{{^required}}?{{/required}}.rawValue{{/isContainer}}{{/isEnum}}{{#hasMore}}, {{/hasMore}}
```

Comment:
I believe it is now correct, but I'd address the readability of this fragment in the future. 
What  `isContainer` means by the way?